scripts/core/sampler.yml

```diff
@@ -107,6 +107,42 @@ returns:
     - $X_RESULT_ERROR_OUT_OF_RESOURCES
 --- #--------------------------------------------------------------------------
 type: function
+desc: "Create a sampler object in a context"
+class: $xSampler
+name: CreateExp
```

Comment:
I would recommend using the `pNext` chaining to pass additional arguments to `urSamplerCreate`

Reply:
Done. The new struct is `$x_exp_sampler_mip_properties_t`